packages/manager/modules/pci/src/projects/project/training/jobs/translations/Messages_fr_FR.json

```diff
@@ -39,5 +39,11 @@
   "pci_projects_project_training_jobs_submit_data_permission_RO": "Lecture Seule",
   "pci_projects_project_training_jobs_submit_data_permission_RW": "Lecture & Ecriture",
   "pci_projects_project_training_jobs_submit_ovhai_info": "La commande suivante vous permet de lancer le même job grâce à notre CLI ovhai :",
-  "pci_projects_project_training_jobs_submit_ovhai_help": "Pour plus d'information sur les paramètres disponibles :"
+  "pci_projects_project_training_jobs_submit_ovhai_help": "Pour plus d'information sur les paramètres disponibles :",
+  "pci_projects_project_training_jobs_id": "Job ID",
```

Comment:
Are you expecting that `Job ID` to be translated in other languages?
If not, then you should remove it from here.